src/mars.d

```diff
@@ -951,6 +951,11 @@ Language changes listed by -transition=id:
                     global.params.objname = p;
                     continue;
                 }
+                if (strcmp(p, `/?`) == 0)
```

Comment:
No - the lines above follow the starting `if` statement which checks whether there is a `-`.
Also this flag is only added for windows.